srt.py

```diff
@@ -18,11 +18,14 @@
 # accept it, so we do too.
 RGX_TIMESTAMP_MAGNITUDE_DELIM = r"[,.:，．。：]"
 RGX_TIMESTAMP = RGX_TIMESTAMP_MAGNITUDE_DELIM.join([r"\d+"] * 4)
+RGX_TIMESTAMP_PARSEABLE = RGX_TIMESTAMP_MAGNITUDE_DELIM.join(
```

Comment:
I suppose if we are going to do this, we could just make it part of `SRT_REGEX` -- is there a problem I'm missing with that (ie. just unifying with `RGX_TIMESTAMP`)?

Reply:
Ah, and re: my comment about avcodec below, I guess let's actually allow arbitrarily sized msec

Reply:
@cdown if we unify with `RGX_TIMESTAMP`, then when we're iterating over the matches, it will look a bit messier, something like this maybe:

```
...
raw_index, start_hrs, start_mins, start_secs, start_msecs, end_hrs, end_mins, end_secs, end_msecs, proprietary, content = match.groups()
...
        yield Subtitle(
            index=int(raw_index),
            start=srt_timestamp_to_timedelta(start_hrs, start_mins, start_secs, start_msecs),
            end=srt_timestamp_to_timedelta(end_hrs, end_mins, end_secs, end_msecs),
            content=content.replace("\r\n", "\n"),
            proprietary=proprietary,
        )
```

In the next iteration of the PR, I'm constructing `RGX_TIMESTAMP` and `RGX_TIMESTAMP_PARSEABLE` from the same building blocks to avoid code duplication while still getting the slightly nicer code cleanliness.